index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>My First Page</title>
+</head>
+<body>
+    <h1>My First Page</h1>
+    <p><a href="../about/index.html">Об Авторе</a></p>   
```

Comment:
"../" means that you  move up  1 level in your hierarchy. At the moment, your file is already in the root of the repository, and moving up it even higher by 1 level is unnecessary, see my previous comment on this file